src/material.js

```diff
@@ -162,20 +164,6 @@ export class Material {
         });
     }
 
-    set Basis(config) {
-        const newConfig = {
-            elements: config.elements,
-            coordinates: config.coordinates,
-            units: config.units,
-            cell: config.cell,
-            isEmpty: false
-        };
-        return new ConstrainedBasis({
-                ...newConfig
-            }
-        );
-    }
-
```

Comment:
This function ended up not being needed after updating the function `convertMDMaterialToMaterial(material)` in the web-app (https://github.com/Exabyte-io/exabyte-web-app/pull/1663/files#)